miasm2/os_dep/win_32_structs.py

```diff
@@ -0,0 +1,185 @@
+from miasm2.core.types import MemStruct, Num, Ptr, Str, \
+    Array, RawStruct, Union, \
+    BitField, Self, Void, Bits, \
+    set_allocator, MemUnion, Struct
+
+
+"""
+kd> dt nt!_LDR_DATA_TABLE_ENTRY
++0x000 InLoadOrderLinks : _LIST_ENTRY
++0x008 InMemoryOrderLinks : _LIST_ENTRY
++0x010 InInitializationOrderLinks : _LIST_ENTRY
++0x018 DllBase : Ptr32 Void
++0x01c EntryPoint : Ptr32 Void
++0x020 SizeOfImage : Uint4B
++0x024 FullDllName : _UNICODE_STRING
++0x02c BaseDllName : _UNICODE_STRING
++0x034 Flags : Uint4B
++0x038 LoadCount : Uint2B
++0x03a TlsIndex : Uint2B
++0x03c HashLinks : _LIST_ENTRY
++0x03c SectionPointer : Ptr32 Void
++0x040 CheckSum : Uint4B
++0x044 TimeDateStamp : Uint4B
++0x044 LoadedImports : Ptr32 Void
++0x048 EntryPointActivationContext : Ptr32 Void
++0x04c PatchInformation : Ptr32 Void
+"""
+
+
+class UnicodeString(MemStruct):
+    fields = [
+        ("length", Num("H")),
+        ("maxlength", Num("H")),
+        ("data", Ptr("<I", Str("utf16"))),
+    ]
+
+
+class ListEntry(MemStruct):
+    fields = [
+        ("flink", Ptr("<I", Void())),
+        ("blink", Ptr("<I", Void())),
+    ]
+
+
+class LdrDataEntry(MemStruct):
+    fields = [
+        ("InLoadOrderLinks", ListEntry),
+        ("InMemoryOrderLinks", ListEntry),
+        ("InInitializationOrderLinks", ListEntry),
+        ("DllBase", Num("<I")),
+        ("EntryPoint", Num("<I")),
+        ("SizeOfImage", Num("<I")),
+        ("FullDllName", UnicodeString),
+        ("BaseDllName", UnicodeString),
+        ("flags", Array(Num("B"), 4)),
+        ("LoadCount", Num("H")),
+        ("TlsIndex", Num("H")),
+        ("union1", Union([
+            ("HashLinks", Ptr("<I", Void())),
+            ("SectionPointer", Ptr("<I", Void())),
+        ])),
+        ("CheckSum", Num("<I")),
+        ("union2", Union([
+            ("TimeDateStamp", Num("<I")),
+            ("LoadedImports", Ptr("<I", Void())),
+        ])),
+        ("EntryPointActivationContext", Ptr("<I", Void())),
+        ("PatchInformation", Ptr("<I", Void())),
+
+    ]
+
+"""
```

Comment:
Can you move these strings into the associated class? That way, it will be a docstring, easier to use / retrieve